lib/net.js

```diff
@@ -760,7 +760,7 @@ function afterWrite(status, handle, req, err) {
   }
 
   if (status < 0) {
-    var ex = exceptionWithHostPort(status, 'write', req.address, req.port);
+    var ex = errnoException(status, 'write', req.error);
```

Comment:
There is no `req.address` or `req.port` on `write`. I think it has been put there mistakingly.

The only difference that is going to be is the absence of `err.address`, which was previously `undefined`. Therefore I consider this a bugfix.